### ISSUE TITLE
Can't open in Mac

### ISSUE THREAD

**Issue by dealancer** [nwoolls/MultiMiner#275]
```
➜  ~ open /Applications/MultiMiner.app
LSOpenURLsWithRole() failed with error -10810 for the file /Applications/MultiMiner.app.
```

**Comment by jseliger1** [comment]
I had this same problem. Try showing package contents and opening MultiMiner.win manually: https://github.com/nwoolls/MultiMiner/issues/68

**Comment by iliashenko** [comment]
Same problem.

**Comment by tommaton** [comment]
+1 - same problem here

**Comment by mnifakram** [comment]
+1 same problem

**Comment by nwoolls** [comment]
I've updated the installation instructions here with a workaround:

* https://github.com/nwoolls/MultiMiner/wiki/Installation#mac-os-x

```
cd ~/Desktop/MultiMiner.app/Contents/Resources
mono --arch=32 MultiMiner.Win.exe
```

**Comment by nwoolls** [comment]
Fixed in [v4.1.1](https://github.com/nwoolls/MultiMiner/releases/tag/v4.1.1) - thanks for taking the time to report this.
